app/lib/account/backend.dart

```diff
@@ -214,26 +223,22 @@ class AuthenticatedUser implements AuthenticatedAgent {
 
 /// Verifies the current bearer token in the request scope and returns the
 /// current authenticated user or a service agent with the available data.
-Future<AuthenticatedAgent> requireAuthenticatedAgent(
-    {AuthSource? source}) async {
+Future<AuthenticatedAgent> requireAuthenticatedClient() async {
   final token = _getBearerToken();
```

Comment:
Can't we make a `requireAuthenticatedAgent` that is the only thing that calls `_getBearerToken`?

Reply:
Planning to do in one of the next steps. I can add it to this PR, but I'm afraid it would be harder to verify the changes.